client/src/App.js

```diff
@@ -82,18 +89,21 @@ function App() {
         <Route
           exact
           path="/login"
-          element={
-            isLoggedIn ? (
-              <Navigate replace to="/" />
-            ) : (
-              <Login checkUser={checkUser} />
-            )
-          }
+          Component={() => {
+            if (isLoggedIn) {
+              return <Navigate replace to="/" />;
+            } else {
+              // <Login checkUser={checkUser} />
```

Comment:
remove comments

Reply:
+ login component is not needed. Delete the file

Reply:
resplved